client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -55,20 +51,22 @@ class EditorRevisionsList extends PureComponent {
 		KeyboardShortcuts.on( 'move-selection-down', this.selectPreviousRevision );
 
 		if ( ! this.props.selectedRevisionId ) {
-			this.trySelectingInterval = setInterval( this.trySelectingFirstRevision, 500 );
```

Comment:
Also nice :) I think I'd drafted a comment a while back about how this could have side effects but had forgot to actually hit go on that one.